system/default_ad_api/include/default_ad_api/specs/interface/version.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DEFAULT_AD_API__SPECS__INTERFACE__VERSION_HPP_
+#define DEFAULT_AD_API__SPECS__INTERFACE__VERSION_HPP_
+
+#include "autoware_ad_api_msgs/srv/interface_version.hpp"
+
+namespace ad_api::interface::version
+{
+
+struct T
+{
+  using Service = autoware_ad_api_msgs::srv::InterfaceVersion;
+  static constexpr char name[] = "/api/v0/interface/version";
```

Comment:
Why is this `interface/version`, not `interface_version` or just `version`?
Are there any other APIs planned like `interface/foo`?

Reply:
Yes. For example, API list of supported older versions or information to check binary compatibility of messages. And clarify the difference from the released version of Autoware.

Reply:
Okay, thank you. Considering other APIs will have a category, then I think it's good to have a category for the version API to be consistent.